content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -451,19 +451,17 @@ customize policies to individual workloads, and secondly, it may be challenging
 single level of "tenancy" that should be given a namespace. For example, an organization may have
 divisions, teams, and subteams - which should be assigned a namespace?
 
-To solve this, Kubernetes provides the [Hierarchical Namespace Controller (HNC)](https://github.com/kubernetes-sigs/hierarchical-namespaces),
-which allows you to organize your namespaces into hierarchies, and share certain policies and
-resources between them. It also helps you manage namespace labels, namespace lifecycles, and
-delegated management, and share resource quotas across related namespaces. These capabilities can
-be useful in both multi-team and multi-customer scenarios.
```

Comment:
I would separate out a description of the approach (with HNC mention dropped) from mention of third-party projects, and hoist the `{{% thirdparty-content %}}` above the individual section headers:

```
One possible approach is to organize your namespaces into hierarchies, and share certain policies and
resources between them. This could include managing namespace labels, namespace lifecycles,
delegated access, and shared resource quotas across related namespaces. These capabilities can
be useful in both multi-team and multi-customer scenarios.

Some third-party projects that provide similar capabilities and aid in managing namespaced resources are
listed below.

{{% thirdparty-content %}}
```